src/DedicatedHsm/DedicatedHsm.sln

```diff
@@ -13,7 +13,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AuthenticationAssemblyLoadC
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Authenticators", "..\Accounts\Authenticators\Authenticators.csproj", "{467DE3DA-2D49-444B-933B-A01E621F0000}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Az.DedicatedHSM", "Az.DedicatedHSM.csproj", "{639F2669-5F10-418C-B5C6-768D04A30ECA}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Az.DedicatedHsm", "Az.DedicatedHsm.csproj", "{639F2669-5F10-418C-B5C6-768D04A30ECA}"
```

Comment:
This change is not related to the alias; but to fix a build error on GH codespace (linux)